pmultiqc/modules/quantms/maxquant.py

```
@@ -0,0 +1,1135 @@
+
+def read(file_path, file_type=None, filter_type=None):
+    mq_data = pd.read_csv(file_path, sep="\t", low_memory=False)
+
+    if filter_type == "Reverse":
+        # Reverse
+        # When marked with ‘+’, this particular protein group contains no protein,
+        # made up of at least 50% of the peptides of the leading protein, with a
+        # peptide derived from the reversed part of the decoy database.
+        # These should be removed for further data analysis.
+        # The 50% rule is in place to prevent spurious protein hits to erroneously flag the protein group as reverse.
+        if "Reverse" in mq_data.columns:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)